recipes/trie/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "trie" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trie-{{ version }}.tar.gz
+  sha256: 0729a159ddef4bfa4c05478017b93a7a1af4e65e8a7b7628676bad13199ffe39
+
+build:
+  skip: true  # [py>=310]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools-scm
+  run:
+    - python
+    - eth-hash >=0.1.0,<1.0.0
+    - eth-utils >=2.0.0,<3.0.0
+    - hexbytes >=0.2.0
+    - rlp >=3,<4
+    - sortedcontainers >=2.1.0,<3
+    - typing-extensions >=4.0.0,<5  # [py<38]
```

Comment:
```suggestion
```
This keeps in noarch. Conda-forge only supports python>=3.8 for non noarch packages so we can just restrict it.